File: BackEnd/ajax/Cirugia.js
$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarCirugia',
    'idIngreso': null,
    'idTipoCirugia': null,
    'idMedico': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorHoy',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipoCirugia',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroFecha',
    'idCentroMedico': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarTipoCirugia',
    'descripcion': null,
    'idPiso': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorMedico',
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorMedicoFecha',
    'idMedico': null,
    'fechaHora': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroMedico',
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Cirugia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTipoCirugia',
    'descripcion': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/Cirugia.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Cirugia'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'agregarCirugia',
  'fechaHora': None,
  'idTipoCirugia': None,
  'idIngreso': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'listarPorHoy',
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'listarTipoCirugia',
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'listarPorCentroFecha',
  'idCentroMedico': None,
  'fechaHora': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'actualizarTipoCirugia',
  'descripcion': None,
  'idPiso': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'listarPorMedico',
  'idMedico': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'listarPorMedicoFecha',
  'fechaHora': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'listarPorCentroMedico',
  'idCentroMedico': None,
}
print postear(data,url)
print

url='Cirugia'
data={
  'accion':'agregarTipoCirugia',
  'descripcion': None,
}
print postear(data,url)
print
